### ISSUE TITLE
[PR] Close the splashscreen programmatically

### ISSUE THREAD

**Issue by angel1st** [KarimMohamed2005/SplashScreenFlutterPackage#20]
There are certain use cases, when the splash screen should stay until the app is actually ready e.g. load DB, settings, connect to the backend and etc. In such cases, the splash screen should be removed from the app, once everything is ready and this moment might vary on different devices. Hence instead of `seconds` property, it would be nice to have a method e.g. `hide`, which hides the splash screen programmatically. In case such method exists, please let me know, otherwise, I believe it would be worthwhile to have it implemented.

**Comment by terryBaz** [comment]
I already had this feature implemented in my local clone, **but it will be there some breaking changes**, let me know if you want, i could prepare a PR.

**Comment by angel1st** [comment]
@terryBaz - I think this feature is needed, it would be nice to have a PR, thanks!

**Comment by wuestkamp** [comment]
Any news on this? I would also like to disable the SplashScreen by touch!
